code/modules/antagonists/heretic/knowledge/moon_lore.dm

```diff
@@ -0,0 +1,262 @@
+		for where the Ringleader had started the parade, I shall continue it unto the suns demise \
+		WITNESS MY ASCENSION, THE MOON SMILES ONCE MORE AND FOREVER MORE IT SHALL!"
+	route = PATH_MOON
+
+/datum/heretic_knowledge/ultimate/moon_final/is_valid_sacrifice(mob/living/sacrifice)
+
+	var/brain_damage = sacrifice.get_organ_loss(ORGAN_SLOT_BRAIN)
+	// Checks if our target has enough brain damage
+	if(brain_damage < 50)
+		return FALSE
+
+	return ..()
+
+/datum/heretic_knowledge/ultimate/moon_final/on_finished_recipe(mob/living/user, list/selected_atoms, turf/loc)
+	. = ..()
+	priority_announce("[generate_heretic_text()] Laugh, for the ringleader [user.real_name] has ascended! \
+					The truth shall finally devour the lie! [generate_heretic_text()]","[generate_heretic_text()]", ANNOUNCER_SPANOMALIES)
+
+	user.client?.give_award(/datum/award/achievement/misc/moon_ascension, user)
+	ADD_TRAIT(user, TRAIT_MADNESS_IMMUNE, REF(src))
+
+	RegisterSignal(user, COMSIG_LIVING_LIFE, PROC_REF(on_life))
+
+	// Roughly 1/5th of the station will rise up as lunatics to the heretic
+	for (var/mob/living/carbon/human/crewmate as anything in GLOB.human_list)
+		// How many lunatics we have
+		var/amount_of_lunatics = 0
+		if (isnull(crewmate.mind))
+			continue
+		// Heretics, lunatics and monsters shouldn't become lunatics because they either have a master or have a mansus grasp
+		if (IS_HERETIC_OR_MONSTER(crewmate))
+			to_chat(crewmate, span_boldwarning("[user]'s rise is influencing those who are weak willed. Their minds shall rend." ))
+			continue
+		// Mindshielded and anti-magic folks are immune against this effect because this is a magical mind effect
+		if(HAS_TRAIT(crewmate, TRAIT_MINDSHIELD) || crewmate.can_block_magic(MAGIC_RESISTANCE))
+			to_chat(crewmate, span_boldwarning("You feel shielded from something." ))
+			continue
+		if(amount_of_lunatics > length(GLOB.human_list) / 5)
+			to_chat(crewmate, span_boldwarning("You feel uneasy, as if for a brief moment something was gazing at you." ))
+			continue
+		var/datum/antagonist/lunatic/lunatic = crewmate.mind.add_antag_datum(/datum/antagonist/lunatic)
+		lunatic.set_master(user.mind, user)
+		var/obj/item/clothing/neck/heretic_focus/moon_amulette/moon_amulette = new
+		crewmate.put_in_active_hand(moon_amulette)
+		crewmate.emote("laugh")
+
+/datum/heretic_knowledge/ultimate/moon_final/proc/on_life(mob/living/source, seconds_per_tick, times_fired)
+	SIGNAL_HANDLER
+
+	visible_hallucination_pulse(
+		center = get_turf(source),
+		radius = 7,
+		hallucination_duration = 60 SECONDS
+	)
+
+	for(var/mob/living/carbon/carbon_view in view(7, source))
+		var/carbon_sanity = carbon_view.mob_mood.sanity
+		if(IS_HERETIC_OR_MONSTER(carbon_view))
+			continue
+		carbon_view.adjust_confusion(2 SECONDS)
+		carbon_view.mob_mood.set_sanity(carbon_sanity - 5)
+		if(carbon_sanity < 30)
+			to_chat(carbon_view, span_warning("you feel your mind begining to rend!"))
```

Comment:
```suggestion
			to_chat(carbon_view, span_warning("you feel your mind beginning to rend!"))
```
Also, are we sure we want the message to be sent each and every life cycle? It's going to be obnoxious.